### DISCUSSION TITLE
Panjabi playlist

### DISCUSSION THREAD

**Discussion by random12799** [iptv-org/community#1086] [Q&A]
Hi, I noticed that the Panjabi playlist has 27 channels but the channel.cvs shows 48 was wondering how could I just add those to the playlist.
Thanks

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/FAQ.md#why-is-the-channel-on-the-iptv-orggithubio-but-not-in-the-playlist
